server/models/contest.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const contestSchema = new Schema({
```

Comment:
My `user` model uses uppercase like `UserSchema` and filename `User.js`. For consistency, can we agree to do it that way? Super minor thing but figured I'd mention it :)

Reply:
No worries, consistent code will help us.